fe-games/src/index.css

```diff
@@ -117,7 +113,21 @@ justify-content: right;
   padding: 3rem;
   text-align: center;
 }
-
+.comment_list{
+   display: flex;
+   flex-wrap: wrap;
+   align-items: space-between;
+   justify-content: center;
+   margin: 0 auto;
+   max-width: 1200px;
+}
+.comment {
+  background-color: rgba(0, 0, 0, 0.1);
+  padding: 20px 25px;
+  border-radius: 18px;
+  font-size: 0.8em;
+  margin: 15px;
+}
 .review .img-container {
```

Comment:
When viewing your app in the browser, I wasn't able to see the text for any of the elements. It seems that the native colour setting is affecting this. I'd suggest utilising the class names you have set for comment_body, comment_author, comment_votes to style them in your CSS file.